app/main.py

```diff
@@ -2,24 +2,66 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, instance: "Visitor", name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: "Visitor", owner: object) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: "Visitor", value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError()
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError()
```

Comment:
Same as above, it's better to provide an error message when raising exceptions. Modify this line to `raise ValueError('Value out of the allowed range')`.